src/sentry/interfaces/message.py

```diff
@@ -61,7 +61,7 @@ def to_python(cls, data):
 
         if kwargs['formatted']:
             if not isinstance(kwargs['formatted'], six.string_types):
-                data['formatted'] = json.dumps(data['formatted'])
+                kwargs['formatted'] = json.dumps(data['formatted'])
         # support python-esque formatting (e.g. %s)
```

Comment:
So this never worked? Do we want to do this then instead of just deleting the code?

Reply:
Would love to kill that, actually! Could someone else please confirm, just to be safe?

Reply:
Yeah, I don't see how this could have handled anything before so it seems like we can drop it.